ERC20 transfer tutorial added, file name consistency implemented, README.md edited.

blackwidoq opened the pull request:
Some minor changes are made to `avax-c-chain-to-eth-address-transfer-tutorial.md`
A new tutorial `ERC20-token-transfer-to-ETH-address.md` added, as proposed.
`README.md` in `avalanche` folder has been edited accordingly.

yash-sharma1 reviewed (changes requested):
@blackwidoq Appreciate it, very well written and this is a really improvised version of your work. 
- All things are good just one thing which needs to be added that is after every execution of the code make sure to add the sample output so user/learners can get an idea that what should be the output and if they run in any issue it'll help them in resolving that.
- Before the next commit make sure to edit the `SUMMARY.md` file and add your new tutorial in that. For more details refer to this guide (Last Step) [**HERE**](https://docs.google.com/document/d/13LWLrWzZ34M0ldWGeDANcWxw9nEWk3AX3VwXRBIOs1M/edit?usp=sharing)

blackwidoq commented:
@yash-sharma1  Hello, ive made the changes requested. I've added cropped images of the terminal outputs and added some comments. Also, SUMMARY.md has been changed.

yash-sharma1 commented on network-documentation/avalanche/tutorials/avax-c-chain-to-eth-address-transfer-tutorial.md:
```diff
@@ -2,57 +2,68 @@
 description: Teaching how to Transfer AVAX native tokens from C chain to ETH address
 ---
 
+## About the author
```
"About the Author" will come after the title of the tutorial and new heading/topic will come as Introduction

yash-sharma1 commented on network-documentation/avalanche/tutorials/avax-c-chain-to-eth-address-transfer-tutorial.md:
```diff
@@ -2,57 +2,68 @@
+This tutorial was created by [Seongwoo Oh](https://github.com/blackwidoq). He is a student and an Avalanche novice.
+
 # Transfer AVAX tokens from the C chain of your AVAX wallet to an ETH address \(Metamask, for example\)
 
```
Mention Introduction here

yash-sharma1 commented on network-documentation/avalanche/tutorials/ERC20_token_transfer_to_ETH_address.md:
```diff
@@ -0,0 +1,209 @@
+
+
+# Transfer ERC-20 tokens from the C-chain of your AVAX wallet to an ETH address \(Metamask, for example\)
+
```
Mention "Introduction" here

yash-sharma1 commented on network-documentation/avalanche/tutorials/ERC20_token_transfer_to_ETH_address.md:
```diff
@@ -0,0 +1,209 @@
+description: Teaching how to Transfer ERC-20 tokens from C-chain to ETH address
+---
+
+## About the author
```
Put it after the heading of the tutorial

blackwidoq commented:
@yash-sharma1  Thank you for the suggestions. I made formatting changes to both scripts.
